src/applications/gi/containers/Modals.jsx

```diff
@@ -571,14 +573,19 @@ export class Modals extends React.Component {
             degree and are getting a teaching certification.
           </p>
           <p>
-            To learn more about the STEM Scholarship,{' '}
+            {environment.isProduction() &&
+              'To learn more about the STEM Scholarship,'}{' '}
+            {!environment.isProduction() &&
```

Comment:
Instead of these dual statements, can we put a ternary here instead?
```jsx
{environment.isProduction()
  ? 'To learn more about the STEM Scholarship,'
  : 'To learn more about this scholarship,'}{' '}
```

(Same with the ones above and below.)

Reply:
Thanks for the feedback @cvalarida, going forward I will implement it that way.